src/modules/postScore.js

```diff
@@ -0,0 +1,17 @@
+export default class Post {
+    static post = async (data = {}) => {
+    //   const apiKey = 'Zl4d7IVkemOTTVg2fUdz';
+    //   const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+    //   const url = `${baseURL}games/${apiKey}/scores/`;
```

Comment:
- [ ] [OPTIONAL] Kindly remove the commented code to make your code cleaner and more professional.